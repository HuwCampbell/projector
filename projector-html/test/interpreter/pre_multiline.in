{|<pre><code>    foo
    bar
  baz
quux
   wom
    lop
    </pre></code>|}<span>code</span>
