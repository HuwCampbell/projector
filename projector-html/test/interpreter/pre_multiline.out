Nested
  [ Element
      "pre"
      []
      (Element
         "code"
         []
         (Nested
            [ Raw "    "
            , Raw "foo"
            , Raw "\n"
            , Raw "    "
            , Raw "bar"
            , Raw "\n"
            , Raw "  "
            , Raw "baz"
            , Raw "\n"
            , Raw "quux"
            , Raw "\n"
            , Raw "   "
            , Raw "wom"
            , Raw "\n"
            , Raw "    "
            , Raw "lop"
            , Raw "\n"
            , Raw "    "
            ]))
  , Element "span" [] (Raw "code")
  ]